### ISSUE TITLE
CompletePurchaseResponse > isSuccessful issue

### ISSUE THREAD

**Issue by ineghi** [ubitransports/omnipay-payzen#16]
Hi guys,

I have a very specific case (using eco vouchers) where the payment status is not set to 'AUTHORISED' but directly to 'CAPTURED' (or at least in a very short period).
Thing is that Payzen is calling the checkUrl url with the status 'CAPTURED' and the payment is isSuccessful() is false.

Could we do something like 

```
public function isSuccessful()
    {
        return $this->getTransactionStatus() == 'AUTHORISED' || $this->getTransactionStatus() == 'CAPTURED';
    } 
```

?

edit: tested locally seems to have no regression AFAICS

**Comment by aschelch** [comment]
Hi, 
We added in the last PR #18 and it's available in version 2.2.0.
Hope it will fix your issue.
I will let you reopen an issue if not.
Best regards !

**Comment by ineghi** [comment]
Works like a charm ! thanks for the quick reaction !
